cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -50,7 +50,7 @@ service PackagesService {
 
   rpc UpdateInstalledPackage(UpdateInstalledPackageRequest) returns (UpdateInstalledPackageResponse) {
      option (google.api.http) = {
-       patch: "/core/packages/v1alpha1/installedpackages"
+       put: "/core/packages/v1alpha1/installedpackages"
```

Comment:
Nice, so there will be no problem with us later providing a `patch` method here either (though I understand the message formats will need references throughout).

Reply:
yes, right on